### PR TITLE
feat: set umeed as default

### PR DESCRIPTION
**Pull request by RafilxTenfen**
- Right now the env variables are set to umeed 
- add gravity gen tx to make it work
- Now you can run `make setup-chain`

> tip, the logs can be seen with `journalctl -fu umeed-1`

### CODE REVIEW

**Comment by toteki** [comment]
Diff itself looks good - btw, do you think it's still iffy to try on macOS?

**Comment by RafilxTenfen** [comment]
> Diff itself looks good - btw, do you think it's still iffy to try on macOS?

I would try to run it on macOS once If you already have all the binary needed by `scripts/deps/prereq.sh` 
or maybe we could remove the go version check and just check for `command_exists go` instead of trying to install '-'

File: scripts/chain/start_chain.sh [hunk 1]
```diff
@@ -120,10 +121,13 @@ else
 fi
 
 echo "INFO: Generating gentxs for validator accounts"
-for (( a=1; a<=$NUM_VALS; a++ ))
+for (( a=2; a<=$NUM_VALS; a++ ))
```
Comment (robert-zaremba) [inline scripts/chain/start_chain.sh:124]: why starting from 2?

Comment (RafilxTenfen) [reply scripts/chain/start_chain.sh:124]: I've hardcoded the first one to generate the gravity-tx

**Review by toteki — APPROVED** [review]

**Pull request merged**